src/services/eventService.ts

```diff
@@ -0,0 +1,36 @@
+import { v4 as uuidv4 } from 'uuid';
+import md5 from 'md5';
+import MeanApiService from './meanApiService';
+import ProviderService from './providerService';
+
+export default class EventService {
+  meanApiService: MeanApiService;
+
+  providerService: ProviderService;
+
+  sessionId: string;
+
+  identifierId: string;
+
+  constructor(meanApiService: MeanApiService, providerService: ProviderService) {
+    this.meanApiService = meanApiService;
+    this.providerService = providerService;
+    this.sessionId = uuidv4();
+  }
+
+  async getIdentifier() {
+    const account = await this.providerService.getAddress();
+
+    return (account && md5(account)) || md5(this.sessionId);
```

Comment:
Weird thing here are:
- If user does not have a wallet connected, and all of a sudden it connects one, then... it's a new session? 
- Every address will look like a unique user (not being able to track DAU/MAU correctly)

Maybe just use sessionId for now? 💀